llarp/config/config.cpp

```diff
@@ -366,6 +366,16 @@ namespace llarp
           m_AuthWhitelist.emplace(std::move(addr));
         });
 
+    conf.defineOption<std::string>(
+        "network",
+        "auth-static",
+        ClientOnly,
+        MultiValue,
+        Comment{
+            "manually add a static auth code to accept for endpoint auth",
```

Comment:
Should this be "... for exit authentication" rather than "endpoint auth"?

Reply:
Also (I think): += ".  Can be specified multiple times"

Reply:
no, it's for exit or non exit traffic authentication.